proto/tendermint/types/types.proto

```diff
@@ -220,6 +220,7 @@ message ShareProof {
 
 // RowProof is a Merkle proof that a set of rows exist in a Merkle tree with a
 // given data root.
+// the row range is end exclusive.
```

Comment:
[nit] capitalize the start of a sentence. Also could combine with the previous line:

```suggestion
// RowProof is a Merkle proof that a set of rows exist in a Merkle tree with a
// given data root. The row range is end exclusive.
```